packages/flutter/lib/src/services/binding.dart

```diff
@@ -47,6 +49,11 @@ mixin ServicesBinding on BindingBase {
     return const _DefaultBinaryMessenger._();
   }
 
+  /// Handler for flutter/system message channel.
```

Comment:
Handler called for messages received on the [SystemChannels.system] call.

Other bindings may override this to respond to incoming system message.